CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+# Introduction
+There are multiple ways to help make TimescaleDB better. All of our
+documentation and source for the PostgreSQL extension are available to use and
+review with GitHub.
+
+## First contribution
+You can make contributions to the documentation by creating a fork of the
+repository.
+
+
+### Procedure: Contributing using a fork
+1.  Make sure you have a [github](github.com) account, and that you're signed in.
+1.  Navigate to the [Timescale Documentation
+Repo](https://github.com/timescale/docs) click the `Fork` button in the
+top-right corner, and select the account you want to use.
+1.  Wait for GitHub to create your fork and redirect you.
+1.  Clone the repository to your local machine. To find this URL, click the green
+    `Code` button and copy the HTTPS URL:
+
+    ```bash
+    git clone https://github.com/<username>/docs.git
+    ```
+
+1.  Change into the directory that contains the repo, and check out the ``/latest`` branch:
```

Comment:
Latest is the default so cloning the repo will give you latest

Reply:
+1